analyzers/src/ITs.JsonParser/Program.cs

```diff
@@ -0,0 +1,40 @@
+﻿using CommandLine;
+using ITs.JsonParser;
+
+Parser.Default.ParseArguments<CommandLineOptions>(args).MapResult(Execute, errs => 1);
+
+int Execute(CommandLineOptions options)
+{
+    WriteLineColor("Processing analyzer results", ConsoleColor.Yellow);
+    NewIssueReports();
+    ShowDiffResults(options.Project, options.RuleId);
+    return 0;
+}
+
+// TODO: Reimplement this functionality from the powershell function.
+void NewIssueReports()
+{
+    Console.WriteLine("Normalizing the SARIF reports");
```

Comment:
..to go clean already, you should also change this to something more meaningful

Reply:
Almost the entire file is going to be re-written, and the structure will change.
For now I just want to see that the console-writes are executed, without adding any functionality to keep the scope small.